nilearn/glm/tests/test_utils.py

```diff
@@ -16,6 +20,9 @@
                                 get_design_from_fslmat, multiple_fast_inverse,
                                 multiple_mahalanobis, parse_bids_filename,
                                 positive_reciprocal, z_score)
+from nilearn.glm.first_level import FirstLevelModel
+from nilearn.datasets import fetch_spm_auditory
```

Comment:
you can't use `fetch_spm_auditory` in tests; they won't be able to download anything (this is by design, to avoid failures due to data servers' failures, and to avoid slowing down tests with long downloads). you have to use data that you generate (see if anything helps in the `data_gen` module) or that is already stored in the package itself (such as the mni template)

Reply:
Ah!... I see. It is by design not possible to fetch data in test functions. I was not aware of that.

Reply:
I believe it was always preferred not to download data but it only became systematically enforced recently in #2514